smoke-tests/smoke-e2e.bats

```diff
@@ -200,5 +201,14 @@ mk_diag_attr() {
     result=$(attributes_from_span_named "@honeycombio/instrumentation-uikit" viewDidAppear | \
          jq "select (.key == \"className\")" | \
          jq "select (.value.stringValue == \"SwiftUI.UIKitTabBarController\").value.stringValue" | uniq -c)
-    assert_equal "$result" '   5 "SwiftUI.UIKitTabBarController"'
+    assert_equal "$result" '   6 "SwiftUI.UIKitTabBarController"'
+}
+
+@test "Navigation attributes are correct" {
+    result=$(attribute_for_span_key "@honeycombio/instrumentation-navigation" Navigation CurrentNavigationPath string | sort | uniq -c)
+    assert_equal "$result" '  10 "Core"
+   1 "Network"
+   1 "View Instrumentation"
+   1 "[]"
+   1 "[{\"name\":\"Yosemite\"}]"'
```

Comment:
this feels fragile to me because the more we navigate around in our tests, the more spans we'll get here. Perhaps I should be filtering it down and checking that `"[]"` and `"[{\"name\":\"Yosemite\"}]"'` exist at least once?

Reply:
What I ended up with for the other PR was extracting a whole span, like this:

```
# Spans on a particular view.
 # Arguments:
 #   $1 - scope
 #   $2 - span name
 #   $3 - view.name
 spans_on_view_named() {
     spans_received | jq ".scopeSpans[] \
         | select(.scope.name == \"$1\").spans[] \
         | select (.name == \"$2\") as \$span \
         | .attributes?[]? \
         | select (.key? == \"view.name\" and .value.stringValue == \"$3\") \
         | \$span"
 }
```

And then individually picking out the attributes, like this:

```
    class=$(echo "$span" | jq '.attributes[] | select(.key == "view.class").value.stringValue')
    assert_equal "$class" '"UIButton"'
```

Maybe something similar would work here?

Reply:
I think for this particular test, all I want is "did the navigation spans get triggered when we navigate to the screen" and I can do that in a less fragile way by grepping for the navigation lines I want. 

But I can use this method on a different set of spans to ensure, ex. the render spans include the right attribute